_locales/de/messages.json

```diff
@@ -216,7 +220,7 @@
         "description": "Erklärung zum Ersetzen von Text überall auf der Seite"
     },
     "Replacing_anywhere_p1": {
-        "message": "Um Text zu ersetzen, der nicht in einem Texteditor enthalten ist und sich nicht in einem Textfeld befindet, deaktivieren Sie die Kontrollkästchen <code>Nur Eingabefelder</code> und <code>Nur sichtbaren Inhalt ändern</code>.",
```

Comment:
Why did this change?

Reply:
The original functionality and the functionality of the new extension are slightly different.
So I updated the help section and I changed this.